src/components/Dashboard/TodoList.vue

```diff
@@ -1147,8 +1190,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin) {
           await this.loadSpecialResolution(task)
           break
         default:
-          // eslint-disable-next-line no-console
-          console.log('ERROR - invalid name in filing header =', header)
+          await this.loadDefaultTask(task)
```

Comment:
There are some tasks that appear in the Todo List and that we don't handle. This new code will actually display them, making it clear to Ops what needs to be fixed (ie, unblock task in Todo List).